api/BUILD

```diff
@@ -73,6 +73,7 @@ proto_library(
     visibility = ["//visibility:public"],
     deps = [
         "//contrib/envoy/extensions/compression/qatzip/compressor/v3alpha:pkg",
+        "//contrib/envoy/extensions/compression/qatzstd/compressor/v3alpha:pkg",
```

Comment:
This seems a fairly reasonable use case so I don't anticipate debate on whether this is in contrib.

Can you submit two GitHub users as CODEOWNERS in this PR for this extension?

Ideally they will also do the initial review here of everything under `contrib/`.

Reply:
Sure, there are already two users for `/contrib/qat` in CODEOWNERS.
```
/contrib/qat/ @giantcroc @soulxu
```
@soulxu Hi, could you help to review code? Thanks!

Reply:
Yea, I will help on the review, thanks!